dynatrace-operator-google-marketplace/dockerfile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Dynatrace LLC
```

Comment:
There are a few ocurrences of this:
```suggestion
# Copyright 2020 Dynatrace LLC
```

I suggest also renaming the file to Dockerfile (uppercase D)

Reply:
Done